packages/mdc-theme/_variables.scss

```diff
@@ -144,3 +150,7 @@ $mdc-theme-property-values: (
 @function mdc-theme-is-valid-theme-prop-value_($style) {
   @return type-of($style) == "color" or $style == "currentColor" or str_slice($style, 1, 4) == "var(";
 }
+
+@function mdc-theme-text-emphasis($emphasis) {
```

Comment:
This seems like something we should PR as feat(theme) on master then water into this branch, so it can have its own changelog item.

Also, let's make the number formatting consistent (remove the leadaing 0 on high/medium values).

Reply:
Created PR #3411 for this. This diff will be removed when other PR gets into master and once we water it .